Description: Fix typo: s/AccesDenied/AccessDenied/
Author: Filip Noetzel <filip@j03.de>
Bug: https://github.com/nicolargo/glances/pull/154
Applied-Upstream: https://github.com/nicolargo/glances/commit/ebf8cf9
Last-Update: 2012-12-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/glances/glances.py
+++ b/glances/glances.py
@@ -507,19 +507,19 @@
 
         try:
             procstat['memory_info'] = proc.get_memory_info()
-        except psutil.AccesDenied:
+        except psutil.AccessDenied:
             procstat['memory_info'] = {}
 
         try:
             procstat['memory_percent'] = proc.get_memory_percent()
-        except psutil.AccesDenied:
+        except psutil.AccessDenied:
             procstat['memory_percent'] = {}
 
         if psutil_get_cpu_percent_tag:
             try:
                 procstat['cpu_percent'] = \
                     proc.get_cpu_percent(interval=0)
-            except psutil.AccesDenied:
+            except psutil.AccessDenied:
                 procstat['cpu_percent'] = {}
 
         try:
